src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/OperatorOverloadsHaveNamedAlternatesTests.Fixer.cs

```diff
@@ -10,24 +9,402 @@ namespace Microsoft.ApiDesignGuidelines.Analyzers.UnitTests
 {
     public class OperatorOverloadsHaveNamedAlternatesFixerTests : CodeFixTestBase
     {
+        #region Boilerplate
+
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicOperatorOverloadsHaveNamedAlternatesAnalyzer();
+            return new OperatorOverloadsHaveNamedAlternatesAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpOperatorOverloadsHaveNamedAlternatesAnalyzer();
+            return new OperatorOverloadsHaveNamedAlternatesAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicOperatorOverloadsHaveNamedAlternatesFixer();
+            return new OperatorOverloadsHaveNamedAlternatesFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpOperatorOverloadsHaveNamedAlternatesFixer();
+            return new OperatorOverloadsHaveNamedAlternatesFixer();
+        }
+
+        #endregion
+
+        #region C# tests
+
+        [Fact]
+        public void AddAlternateMethod_CSharp()
+        {
+            VerifyCSharpFix(@"
+class C
+{
+    public static C operator +(C left, C right) { return new C(); }
+}
+",
+@"
+class C
+{
+    public static C operator +(C left, C right) { return new C(); }
+
+    public static C Add(C left, C right)
+    {
+        throw new System.NotImplementedException();
```

Comment:
This is way beyond the MVP for this rule, but... Why not return the result of calling the operator?

```
public static C Add(C left, C right)
{
    return left + right;
}
```